app/services/iugu/iugu_subscription.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class Iugu
+  class IuguSubscription
```

Comment:
I had to rename this one because the specs were mixing it with the Subscription model (at `payment#new` request spec)